pages/hpc/jobs.md

```diff
@@ -1,169 +1,257 @@
-       AccrueTime=2022-12-29T10:45:36
-       StartTime=Unknown EndTime=Unknown Deadline=N/A
-       SuspendTime=None SecsPreSuspend=0 LastSchedEval=2022-12-29T14:40:47 Scheduler=Main
-       Partition=compbio AllocNode:Sid=cpn-m22-24:12178
-       ReqNodeList=(null) ExcNodeList=(null)
-       NodeList=(null)
-       NumNodes=1-1 NumCPUs=1 NumTasks=1 CPUs/Task=1 ReqB:S:C:T=0:0:*:*
-       TRES=cpu=1,mem=32000M,node=1,billing=1
-       Socks/Node=* NtasksPerN:B:S:C=1:0:*:* CoreSpec=*
-       MinCPUsNode=1 MinMemoryNode=32000M MinTmpDiskNode=0
-       Features=(null) DelayBoot=00:00:00
-       OverSubscribe=OK Contiguous=0 Licenses=(null) Network=(null)
-       Command=/panasas/scratch/grp-mygroup/alphafold/CF_Q07283-F1_model1.pl
-       WorkDir=/scratch/12747970/CH_Q07283-F1_model1
-       StdErr=/panasas/scratch/grp-mygroup/alphafold/err_CF_Q07283-F1_model1
-       StdIn=/dev/null
-       StdOut=/panasas/scratch/grp-mygroup/alphafold/out_CF_Q07283-F1_model1
-       Power=
-
-
-    vortex1$ 
-
-The SLURM schduler will try to estimate hte start time of the job - see "StartTime=" in the above output, however it is not always possible to estimate this, particularly if you have multiple queued jobs or there is a downtime scheduled soon and the requested job duration would overrun the start of the downtime.
+## Monitoring Jobs
+
+### Active Jobs monitoring via OnDemand
+CCR offers a detailed view into what is happening on the node(s) where your job is running.  You can view the graphs from OnDemand but you do NOT need to submit the jobs from within OnDemand.  Even jobs submitted from the command line using 'sbatch' are available in your list of Active Jobs.
+
+In OnDemand, click on the arrow next to one of your current jobs and at the bottom you will see two graphs for each node your job is running on.  One graph shows CPU metrics and the other memory (RAM) usage.
+
+
+### Grafana Charts
+
+You can access Grafana charts of your completed jobs, like the Active Jobs available in OnDemand, but you need to query Slurm for the appropriate start and end times and get the node list.  To do this, we provide a script that can be run in the terminal that creates the Grafana URL for your job. 
+
+````
+[ccruser@vortex]$/util/common/metrics/ccr-jobview-url [jobid] [cluster]
+
+
+[ccruser@vortex]$ /util/common/metrics/ccr-jobview-url 10457965 ub-hpc
+````
+
+Then you would paste the outputed link into your browser. For example:
+
+[Click here for example Grafana output](https://dashboard.ccr.buffalo.edu/grafana/d/Vi3oi5gohz/hpc-job-metrics?orgId=1&theme=light&from=1669820527000&to=1669820600000&var-cluster=ub-hpc&var-host=cpn-k08-34-01&var-jobid=10457965)
+
+### Slurm Accounting
+Slurm account information is also available and useful depending on what information you're looking for regarding your jobs.  We provide some suggested commands [here](https://ubccr.freshdesk.com/support/solutions/articles/5000686909-how-to-retrieve-job-history-and-accounting).
+
```

Comment:
Time permitting I would add those couple of sacct & slist commands here as we'll want to delete that page from FD.